core/test/unit/services/slack_spec.js

```diff
@@ -92,68 +92,6 @@ describe('Slack', function () {
         resetSlack();
     });
 
-    describe('makeRequest()', function () {
-        var makeRequest = slack.__get__('makeRequest');
-
-        it('should make request to slack correctly', function () {
-            // set up
-            var reqOptions,
-                pingSlack;
-
-            // fill the options for https request
-            reqOptions = url.parse('https://hooks.slack.com/services/a-b-c-d');
-            reqOptions.method = 'POST';
-            reqOptions.headers = {'Content-type': 'application/json'};
-
-            pingSlack = nock('https://hooks.slack.com/')
-                .post('/services/a-b-c-d', {
-                    text: 'http://myblog.com/mypost',
-                    icon_url: 'http://myblog.com/someImageurl.jpg',
-                    username: 'Ghost'
-                })
-                .reply(200);
-
-            // execute code
-            makeRequest(reqOptions, {
-                text: 'http://myblog.com/mypost',
-                icon_url: 'http://myblog.com/someImageurl.jpg',
-                username: 'Ghost'
-            });
-
-            // assertions
-            pingSlack.isDone().should.be.true();
-        });
-
-        it('can handle an error response correctly', function () {
-            // set up
-            var reqOptions,
-                pingSlack;
-
-            // fill the options for https request
-            reqOptions = url.parse('https://hooks.slack.com/services/a-b-c-d');
-            reqOptions.method = 'POST';
-            reqOptions.headers = {'Content-type': 'application/json'};
-
-            pingSlack = nock('https://hooks.slack.com/')
-                .post('/services/a-b-c-d', {
-                    text: 'http://myblog.com/mypost',
-                    icon_url: 'http://myblog.com/someImageurl.jpg',
-                    username: 'Ghost'
```

Comment:
The value assertions happen already in the ping tests. I don't think we have to keep these tests. Not sure what they actually test except of the assertions. Let me know 👍

Reply:
Did you check coverage?

Reply:
> Did you check coverage?

No, will do.

Reply:
@kirrg001 I can coverage and compared to master, which was 100% coverage for the slack service, it's now on 96.3%, because the error handling [here](https://github.com/TryGhost/Ghost/pull/9335/files#diff-9b04bf9b31d1afb6cca8f6d3baa6540cR62) is not tested anymore.

Reply:
Cool thx, so we can bring back one error case back 👍